share/spack/gitlab/cloud_pipelines/configs/cray-sles/config.yaml

```diff
@@ -0,0 +1,9 @@
+config:
+  concretizer: clingo
+  db_lock_timeout: 120
+  install_tree:
+    root: $spack/opt/spack
```

Comment:
Other than this `root: $spack/opt/spack`, this file appears to be a duplicate of the one already included via `share/spack/gitlab/cloud_pipelines/configs/config.yaml`.

@kwryankrattiger Am I correct that the order of the `--config-scope` arguments to the `spack ci generate ...` command defines the precedence of configs?  So the values here should override the ones in the higher level config I mentioned above?  In that case, this is correct, but could it potentially be reduced to only override the `config:install_tree:root` value?

Reply:
correct, if the only thing that is different here is the `config:install:root` then that is all that needs to be here, it will override since it is a str field and a higher level config.

Reply:
just pushed a commit that removes the config options already defined at the higher scope